Dockerfile

```diff
@@ -45,4 +49,10 @@ RUN cargo install cargo-kcov critcmp cargo-audit cargo-license
 # Install kcov.
 RUN cargo kcov --print-install-kcov-sh | sh
 
+# Install libgpiod (required by vhost-device crate)
+RUN cd /opt/ && \
+    git clone --depth 1 --branch master https://git.kernel.org/pub/scm/libs/libgpiod/libgpiod.git/ && \
```

Comment:
This has the potential to break future builds if the master branch has problems. Can we use a release tag instead?

Reply:
Okay, a release isn't made yet, will update this once the release is out.

Reply:
Do we pause this CI until a release is made or do you want to work on a workaround?

Reply:
Maybe just merge it as is and I will migrate to a release tag once that is available ? It shall happen within a week or two I guess.

Reply:
libgpiod maintainer told me that he will make the release by tomorrow, so I will update the pull request by tomorrow then.

Reply:
If the release is not a viable alternative, than we have a workaround by specifying the exact sha1 to be downloaded. I don't think merging it by using master is a viable choice because then we can run into problems when we want to publish a new container version. We should aim for reproducible builds as much as possible.

Reply:
Updated with the release tag pushed yesterday. Thanks.